code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -215,16 +215,8 @@
 	throw_speed = 3
 	throw_range = 1
 	w_class = WEIGHT_CLASS_BULKY //So you can't hide it in your pocket or some such.
-	var/datum/effect_system/spark_spread/spark_system

```

Comment:
Uhhhh...

Reply:
I suspect its from the blade, the ninja weapon which allows you to slice through cameras.